apps/site/test/site_web/controllers/schedule_controller_test.exs

```diff
@@ -477,15 +477,10 @@ defmodule SiteWeb.ScheduleControllerTest do
 
       assert %Shape{stop_ids: [_ | _] = stop_ids} =
                Enum.find(conn.assigns.route_shapes, &(&1.id == variant))
-
-      assert variant == conn.assigns.active_shape.id
     end
 
-    test "Bus line with correct default shape", %{conn: conn} do
-      conn = get(conn, line_path(conn, :show, "9", "schedule_direction[direction_id]": 1))
-      default_shape_id = List.first(@routes_repo_api.get_shapes("9", direction_id: 1)).id
-      assert conn.assigns.active_shape.id == default_shape_id
-    end
+    # This test was useful on the one hand, but not if the frontend isn't using 
```

Comment:
@hannahpurcell does this comment refer to the code that was removed? If so, might be confusing if left in ;)